### ISSUE TITLE
No Login screen

### ISSUE THREAD

**Issue by mzuenni** [airdcpp-web/airdcpp-webclient#151]
hello since i am only using the webUI localy and the corresponding ports are closed
i wanted to ask if there is an easy way to disable the login authentification and directly be logged in with admin permission (since the password is saved unencrypted localy i dont really need it...)

**Comment by maksis** [comment]
That's not possible at the moment

**Comment by mzuenni** [comment]
is there no easy manipulation in the login script i can do to just return true if asked if an session exists?

**Comment by maksis** [comment]
No, not really. The client will also remove inactive sessions. Possibly the could be an option to allow auto login when connecting from localhost but would require changes to the backend.

**Comment by mzuenni** [comment]
didn't thought it would be hard to change :/
hopefully there will come an option to make this possible either by enabling localhost direct connect or to directly login if no user was specified (what i would prefer)

**Comment by maksis** [comment]
It's probably better to track this in #186

**Comment by maksis** [comment]
Reopening as IP-based auto login still isn't possible

**Comment by StudioEtrange** [comment]
Yes need at least auto login for reverse proxy with security settings !
